--- /scratch/hpc-prf-metdyn/eprop2d1_Jan/53_benchmark_RI_opt/148_Figure4_augDZVP_T2_metdyn/source/src/gw_utils.F	2026-09-20 09:24:34.000000000 +0200
+++ /scratch/hpc-prf-metdyn/eprop2d1_Jan/53_benchmark_RI_opt/424_gw100_tensor_diagnostics_eps15_20261003/ri_regularization_build/gw_utils.F	2026-10-03 21:17:13.000000000 +0200
@@ -1783,10 +1783,9 @@
          CPASSERT(bs_env%auto_ri%ready)
          CALL cp_fm_to_fm(bs_env%auto_ri%M_pq_inv, bs_env%fm_Minv_Gamma)
       ELSE
-         metric_regularization = 0.0_dp
+         metric_regularization = bs_env%regularization_RI
          eigenvalue_threshold = 0.0_dp
          IF (bs_env%do_gw_ri_rs) THEN
-            metric_regularization = bs_env%regularization_RI
             eigenvalue_threshold = bs_env%eps_eigval_mat_RI
          END IF
          CALL RI_2c_integral_mat(qs_env, fm_M, bs_env%fm_RI_RI, bs_env%n_RI, &
